### PR TITLE
Enable a bare Galaxy installation

### PR DESCRIPTION
**Pull request by bgruening**
This will enable Galaxy flavours with no classical Galaxy tool included by default.

### CODE REVIEW

File: templates/startup.sh.j2 [hunk 1]
```diff
@@ -21,6 +21,13 @@ if [ "x$ENABLE_TTS_INSTALL" != "x" ]
         export GALAXY_CONFIG_TOOL_SHEDS_CONFIG_FILE=$GALAXY_HOME/tool_sheds_conf.xml
 fi
 
+# Remove all default tools from Galaxy by default
+if [ "x$BARE" != "x" ]
+    then
+        echo "Remove all tools from the tool_conf.xml file."
+        export GALAXY_CONFIG_TOOL_CONFIG_FILE=config/shed_tool_conf.xml,$GALAXY_HOME/bare_tool_conf.xml
```
Comment (jmchilton) [inline templates/startup.sh.j2]: Where is ``bare_tool_conf.xml`` coming from?

Comment (bgruening) [inline templates/startup.sh.j2]: From a `ADD` statement in a Dockerfile :)

**Comment by bgruening** [comment]
Thanks John.

File: templates/startup.sh.j2 (hunk 1 above)
Comment (jmchilton) [inline templates/startup.sh.j2]: You are your Docker thing :laughing:

Comment (bgruening) [inline templates/startup.sh.j2]: I take this personally ;-)

**Pull request merged**